python/tests/server/test_http_input.py

```diff
@@ -239,10 +243,24 @@ def test_union_integers(client):
 
 
 def test_untyped_inputs():
-    with pytest.raises(TypeError):
-        make_client("input_untyped")
+    config = {"predict": _fixture_path("input_untyped")}
+    app = create_app(
+        config=config,
+        shutdown_event=threading.Event(),
+        upload_url="input_untyped",
+    )
+    assert app.state.health == Health.SETUP_FAILED
+    assert app.state.setup_result.status == schema.Status.FAILED
+    assert app.state.setup_result.logs.find("TypeError: No input type provided for parameter") != -1
```

Comment:
```suggestion
    assert "TypeError: No input type provided for parameter" in app.state.setup_result.logs
```

This would be more idiomatic, but no big deal either way.